android-interop-testing/build.gradle

```diff
@@ -48,26 +48,43 @@ android {
 }
 
 dependencies {
+    implementation 'android.arch.lifecycle:common:1.0.0'
     implementation 'com.android.support:appcompat-v7:26.1.0'
     implementation 'com.android.support:multidex:1.0.3'
     implementation 'com.android.support:support-annotations:27.1.1'
+    implementation 'com.android.support:support-compat:26.1.0'
+    implementation 'com.android.support:support-core-ui:26.1.0'
+    implementation 'com.android.support:support-core-utils:26.1.0'
+    implementation 'com.android.support:support-fragment:26.1.0'
     implementation 'com.google.android.gms:play-services-base:15.0.1'
+    implementation 'com.google.android.gms:play-services-basement:15.0.1'
 
-    implementation project(':grpc-auth'),
+    implementation project(':grpc-api'),
+            project(':grpc-auth'),
             project(':grpc-census'),
+            project(':grpc-context'),
+            project(':grpc-core'),
             project(':grpc-okhttp'),
             project(':grpc-protobuf-lite'),
             project(':grpc-stub'),
             project(':grpc-testing'),
+            libraries.google_auth_credentials,
+            libraries.jsr305,
             libraries.junit,
+            libraries.protobuf_lite,
             libraries.truth
 
     implementation (libraries.google_auth_oauth2_http) {
         exclude group: 'org.apache.httpcomponents'
     }
+    censusApiDependency 'implementation'
     censusGrpcMetricDependency 'implementation'
+    guavaDependency 'implementation'
 
-    compileOnly libraries.javax_annotation
+    compileOnly 'org.hamcrest:hamcrest-core:1.3',
```

Comment:
I'm very concerned to see all these `compileOnly` deps. `compileOnly` deps are very hard to manage. Why are these `compileOnly` instead of `implementation`?

Reply:
With `implementation`, those dependency will be explicitly included (with runtime scope) in pom file. I'm feeling it looks weird. Otherwise I'm okay with `implementation`.

Reply:
Now moved to `implementation`, PTAL.

Reply:
So you had made them compileOnly to make the POM make more sense?

Reply:
Yes. But maybe it did not make more sense.